src/main/java/tempest/Module.java

```diff
@@ -92,12 +92,21 @@ public String toBlankRow() {
     /**
      * Gets the StudySessions related to this module.
      * 
-     * @return The StudySessions related to this module.
+     * @return An array of the StudySessions related to this module.
      */
     public StudySession[] getStudySessions() {
         return studySessions.toArray(new StudySession[0]);
     }
 
+    /**
+     * Gets the StudySessions related to this module.
+     *
+     * @return A list of the StudySessions related to this module.
+     */
+    public LinkedList<StudySession> getStudySessionsList() {
+        return studySessions;
+    }
+
```

Comment:
The idea of getting the study sessions as an array was to simplify the API for the module - do you need this, or could it be done with a transformation using the array?